index.js

```diff
@@ -244,16 +262,15 @@ export const Text = value => Response.Respond (
   Body.Send (value),
 );
 
-//# Json :: Object -> Response a b
+//# Json :: JsonValue -> Response a b
```

Comment:
Does JsonValue mean something like any JSON-serializable data?

Reply:
Yeah. Not all values of type Object are valid input here.